canvas_sdk/commands/__init__.py

```diff
@@ -1,31 +1,41 @@
+from canvas_sdk.commands.commands.allergy import AllergyCommand
 from canvas_sdk.commands.commands.assess import AssessCommand
 from canvas_sdk.commands.commands.diagnose import DiagnoseCommand
+from canvas_sdk.commands.commands.family_history import FamilyHistoryCommand
 from canvas_sdk.commands.commands.goal import GoalCommand
 from canvas_sdk.commands.commands.history_present_illness import (
     HistoryOfPresentIllnessCommand,
 )
 from canvas_sdk.commands.commands.instruct import InstructCommand
+from canvas_sdk.commands.commands.lab_order import LabOrderCommand
 from canvas_sdk.commands.commands.medication_statement import MedicationStatementCommand
+from canvas_sdk.commands.commands.past_surgical_history import PastSurgicalHistoryCommand
 from canvas_sdk.commands.commands.perform import PerformCommand
 from canvas_sdk.commands.commands.plan import PlanCommand
 from canvas_sdk.commands.commands.prescribe import PrescribeCommand
 from canvas_sdk.commands.commands.questionnaire import QuestionnaireCommand
 from canvas_sdk.commands.commands.reason_for_visit import ReasonForVisitCommand
+from canvas_sdk.commands.commands.remove_allergy import RemoveAllergyCommand
 from canvas_sdk.commands.commands.stop_medication import StopMedicationCommand
 from canvas_sdk.commands.commands.update_goal import UpdateGoalCommand
 
 __all__ = (
+    "AllergyCommand",
     "AssessCommand",
     "DiagnoseCommand",
+    "FamilyHistoryCommand",
     "GoalCommand",
     "HistoryOfPresentIllnessCommand",
     "InstructCommand",
+    "LabOrderCommand",
     "MedicationStatementCommand",
```

Comment:
Missing the `MedicalHistoryCommand` here?
```suggestion
    "MedicalHistoryCommand",
    "MedicationStatementCommand",
```